pact/pacts/manage-recalls-ui-manage-recalls-api.json

```diff
@@ -356,6 +356,103 @@
         "headers": {
         }
       }
+    },
+    {
+      "description": "an update recall request",
+      "providerState": "a recall exists",
+      "request": {
+        "method": "PATCH",
+        "path": "/recalls/00000000-0000-0000-0000-000000000000",
+        "headers": {
+          "Authorization": "Bearer accessToken-1"
+        },
+        "body": {
+          "recallLength": "14"
+        }
+      },
+      "response": {
+        "status": 201,
```

Comment:
Pedantry, but 200 or 204 for updated? 201 is created

Reply:
Oh yeah, 204 gets my vote.

Reply:
Yeah I'd already spotted this when updating the pact in the API.  Going for 200, 204 is NO_CONTENT.